packages/contracts-communication/contracts/InterchainClientV1.sol

```diff
@@ -124,8 +126,9 @@ contract InterchainClientV1 is Ownable, InterchainClientV1Events, IInterchainCli
                 executionFee: executionFee,
                 options: options
             });
+            address srcExecutorEOA = IExecutionService(srcExecutionService).executorEOA();
+            IExecutionFees(executionFees).recordExecutor(dstChainId, transactionId, srcExecutorEOA);
```

Comment:
If `srcExecutionService == address(0)`, `recordExecutor()` won't be called for this transaction id, and any associated execution fees could not be claimed. Could either:
- Disallow `srcExecutionService == address(0)` scenario
- Have more than one Recorder in `ExecutionFees`

Reply:
That was fixed in c2271d906833d60643fa1f281665e0e3ee0f2f63